### PR TITLE
Add HTTPS support

### PR DESCRIPTION
**Pull request by paracycle**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -47,11 +49,24 @@ module.exports = function (config) {
...
       var port = addr.port;
-      util.log(util.colors.blue('Server started at http://' + address + ':' + port));
-    });
+      var scheme = 'http' + (config.https ? 's' : '');
+      util.log(util.colors.blue('Server started at ' + scheme + '://' + address + ':' + port));
+    };
+
+    if (config.https) {
+      var opts = {
+        key: fs.readFileSync(config.https.key  || __dirname + '/./ssl/localhost.key'),
```
Comment (nkt) [inline index.js]: Please use `path.join(__dirname, 'ssl/localhost.key')`

Comment (paracycle) [inline index.js]: Sure, done.

**Comment by nkt** [comment]
Great, thanks!

**Comment by nkt** [comment]
Published as `1.2.0`

**Comment by paracycle** [comment]
Thank you so much for the quick turn-around.

**Pull request merged**